massgov/mayflower-patternlab
# replace display initial for IE11 compatibility

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

 IE11 fix for mosaic image not showing up, as well as accordion text being incorrect. 

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-9057)


## Steps to Test


1. View https://mayflower.digital.mass.gov/patterns/05-pages-organization-elected-official/05-pages-organization-elected-official.html# in IE11, ensure that the main image shows up, and on a narrow width, the accordion works as expected.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.
